lib/resource_filter.js

```diff
@@ -142,13 +142,40 @@ module.exports = function (model, filteredKeys) {
     }
   }
 
+  function deleteDeep (item, chunks) {
+    var firstChunk = chunks[0]
+    var firstValue = item[firstChunk]
+
+    if (chunks.length === 1) {
+      delete item[firstChunk]
+    } else if (_.isArray(firstValue)) {
+      firstValue.forEach(function (prop) {
+        deleteDeep(prop, chunks.slice(1))
+      })
+    } else if (_.isObject(firstValue)) {
+      deleteDeep(firstValue, chunks.slice(1))
+    }
+    return item
+  }
+
+  function filterBlacklistedKeys (item, blacklist) {
+    blacklist.forEach(function (path) {
+      var chunks = path.split('.')
+      deleteDeep(item, chunks)
+    })
+    return item
+  }
+
   this.filterObject = function (resource, opts) {
     opts = _.defaults(opts || {}, {
-      access: 'public'
+      access: 'public',
+      blacklist: []
     })
 
     var filtered = filterItem(resource, getExcluded(opts.access, model.modelName))
 
+    filtered = filterBlacklistedKeys(filtered, opts.blacklist)
+
```

Comment:
This should probably be inside resource filter so we don't get too much controller bloat.